eduNEXT/xblock-mindmap
# feat: add functionality for save mind map in s3

### Description

This PR adds the functionality to save each Mind Map in S3.

### How to Test
1. Set the following env variables in your environment:
    ```python
    FILE_UPLOAD_STORAGE_BUCKET_NAME
    AWS_ACCESS_KEY_ID
    AWS_SECRET_ACCESS_KEY
    ```
2. Add the Mind Map component to a unit from Studio
3. Access the unit from LMS and create a new mind map.
4. Click on the "Save" button.
5. Reload the page, you should be the mind map saved.
